threediresultstyler/test/tenbytenraster.asc

```diff
@@ -0,0 +1,19 @@
+NCOLS 10
```

Comment:
Ehm, als je geen tests hebt, kan je dan niet beter de hele test folder weghalen? Nu staat er verwarrend spul in.